docs/reports.rst

```diff
@@ -227,6 +227,9 @@ Redash
 -  api\_key: token to access the query, either for user or for query.
    You can find user's token in the profile, token for query can be
    found in the source page.
+-  refresh: True if you want an updated report. **Important**: For refresh
+   to work the api\_key must be of user type.
+-  parameters: Dictionary of query parameters.
```

Comment:
I'd add a bit some clarification about parameters format: at least the fact that they will have `p_` prepended before making the request.

Reply:
👍